src/m365/tenant/commands/service/service-message-list.ts

```diff
@@ -0,0 +1,96 @@
+import commands from '../../commands';
+import request from '../../../../request';
+import GlobalOptions from '../../../../GlobalOptions';
+import { CommandOption } from '../../../../Command';
+import SpoCommand from '../../../base/SpoCommand';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  workload: string;
+}
+
+class TenantServiceMessageListCommand extends SpoCommand {
+  public get name(): string {
+    return `${commands.TENANT_SERVICE_MESSAGE_LIST}`;
+  }
+
+  public get description(): string {
+    return 'Gets the service messages regarding services in Office 365 from the Office 365 Management API';
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: (err?: any) => void): void {
+    if (this.verbose) {
+      cmd.log(`Getting the service messages regarding services in Office 365 from the Office 365 Management API`);
+    }
+
+    const serviceUrl: string = 'https://manage.office.com/api/v1.0';
+    const statusEndpoint: string = (typeof args.options.workload !='undefined' && args.options.workload) ? `ServiceComms/Messages?$filter=Workload eq '${args.options.workload}'` : 'ServiceComms/Messages';
+
+    this
+      .getTenantId(cmd, this.debug)
```

Comment:
We shouldn't use the `SpoCommand.getTenantId` method to retrieve the tenant ID, because it requires full control SharePoint permissions, which shouldn't be necessary for retrieving message center messages. Instead, we should retrieve the tenant ID from the access token's `tid` property.

Reply:
Can we use `auth.service.tenantId` ?

Reply:
Currently, `auth.service.tenantId` is used only in SPO commands, so in your case it will be still `undefined`, unless you'd first retrieve the tenant ID from the access token and assign it to `auth.service.tenantId` which in this particular case makes little sense since you can retrieve it from the access token directly in the command using the yet to be added util method I mentioned previously.

Reply:
Hi @waldekmastykarz 
Added `Utils.getTenantIdFromAccessToken`
Please review.